### DISCUSSION TITLE
YOLOv4 relu weights

### DISCUSSION THREAD

**Discussion by ngr-francesco** [AlexeyAB/darknet#8906] [Q&A] [answered]
Hello! 
I'm currently working with a quantized version of yolov4 and trying to optimize performance as much as possible. In order to do so I'm considering using yolov4 with relu activations instead of mish (since relu can be easily fused with convolutional and batch normalization layers). In order to do that, though, I need to first get the weights for `yolov4-relu`.
I'm using the coco dataset, on which yolo was trained in the first place, so before resorting to retraining the entire network I thought I'd ask here if anyone has these weights. The links I found in this and in other repos seem to be all broken so I couldn't find them anywhere (also the weights for leaky relu seem to be broken in this repo [YOLOv4-model-zoo](https://github.com/AlexeyAB/darknet/wiki/YOLOv4-model-zoo)). 
If no one has `yolov4-relu` weights, then `yolov4-leaky` would still be useful to get started.
Thanks in advance (and thanks to the devs for maintaining the repo)!

**Comment by stephanecharette** [comment, marked as answer]
1) AlexeyAB is no longer working on this project.  Instead, you should be looking at the new Darknet/YOLO repo here:  https://github.com/hank-ai/darknet#table-of-contents
2) We've been very focused on training performance in the new repo.  The sample network can now be trained in less than 90 seconds:  https://www.youtube.com/watch?v=dq8AVWvWn54
3) See the Darknet/YOLO FAQ for more performance information:  https://www.ccoderun.ca/programming/yolo_faq/#fps

**Reply by ngr-francesco** [reply]
Hi Stephane, thanks for the quick response. That's very helpful!
Then I'll check out that repository, if it's that efficient then hopefully re-training the network with relu activations on the MSCOCO dataset won't be too time consuming. Just out of curiosity: how long would you reckon it will take to train it using Darknet (on a mid/high end GPU)?
PS: If you're interested after training the network I could provide the weights to be added to the repository.
